### PR TITLE
Bugfix for style-align @ jqGrid

### PR DESCRIPTION
**Pull request by kokspflanze**
Hi,

bugfix for https://github.com/ThaDafinser/ZfcDatagrid/issues/220

also translator fix @jqGrid

### CODE REVIEW

File: src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php [hunk 1]
```diff
@@ -36,10 +36,8 @@ private function translate($message)
 
         if (null === $this->translator) {
             if ($this->getServiceLocator()
-                ->getServiceLocator()
```
Comment (ThaDafinser) [inline src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php]: @kokspflanze this didnt worked before?

Comment (kokspflanze) [inline src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php]: the first getServiceLocator method return a [`Zend\ServiceManager\ServiceLocatorInterface`](https://github.com/zendframework/zend-servicemanager/blob/release-2.7.5/src/ServiceLocatorInterface.php#L15), there is no getServiceLocator method, only a get and has-method. thats why i removed the second call.

the same workflow is also in [ZfcDatagrid\Renderer\BootstrapTable\View\Helper\TableRow](https://github.com/ThaDafinser/ZfcDatagrid/blob/master/src/ZfcDatagrid/Renderer/BootstrapTable/View/Helper/TableRow.php#L22)

**Comment by kokspflanze** [comment]
- added unit test for translator
- added phpunit 5
- add comment for align

**Comment by ThaDafinser** [comment]
Thank you!

**Pull request merged**